### PR TITLE
PR for BXMSDOC-6573: Searched for master, slave, blacklist, and whitelist in the RN content and removed it.

### PR DESCRIPTION
**Pull request by kaldesai**
Dedicated JIRA: https://issues.redhat.com/browse/BXMSDOC-6573

- kie-docs/doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.7.0.Final-section.adoc
- kie-docs/doc-content/shared-kie-docs/src/main/asciidoc/Workbench/ReleaseNotes/ReleaseNotesWorkbench.7.11.0.Final-section.adoc

### CODE REVIEW

File: doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.7.0.Final-section.adoc [hunk 1]
```diff
@@ -105,7 +105,7 @@ image:ReleaseNotes/StunnerToolbar-7.0.0.png[align="center", title="Toolbar re-de
 The entire admin console screens have been re-designed focusing on:
 
 ** Create new layout and re-styling existent elements.
```
Comment (hmanwani-rh) [inline doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.7.0.Final-section.adoc:107]:
```suggestion
** Create new layout and re-styling existent elements
```

File: doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.7.0.Final-section.adoc [hunk 2]
```diff
@@ -105,7 +105,7 @@ image:ReleaseNotes/StunnerToolbar-7.0.0.png[align="center", title="Toolbar re-de
...
-** Revise master/detail navigation
+** Revised *Manage Process Instances* page
```
Comment (hmanwani-rh) [inline doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.7.0.Final-section.adoc:108]:
```suggestion
** Revise *Manage Process Instances* page
```

**Review by hmanwani-rh — COMMENTED** [review]
@kaldesai Added a couple of minor comments, rest looks good to me. 
thanks

**Comment by kaldesai** [comment]
@hmanwani-rh , Thank you for reviewing the content I have incorporated your feedback.

**Review by jesuino — APPROVED** [review]
The changes LGTM - thanks!

**Pull request merged**